cmd/cli/kubectl-kyverno/utils/common/common.go

```diff
@@ -486,7 +486,8 @@ OuterLoop:
 		WithNamespaceLabels(namespaceLabels).
 		WithAdmissionInfo(c.UserInfo).
 		WithClient(c.Client).
-		WithSubresourcesInPolicy(subresources)
+		WithSubresourcesInPolicy(subresources).
+		IsMock(true)
```

Comment:
Is `mock` always `true` when executing CLI? What if it has access to the cluster?

Reply:
> Is mock always true when executing CLI?

Yes, it is always `true`.

> What if it has access to the cluster?

Accessing the cluster != Recreating admission review request. `mock` is only used to ignore `subresourceInAdmnReview` when requests are coming from CLI. CLI accesses the cluster to CRUD objects. It cannot populate all the fields of an admission review request. 

So `mock` flag won't interfere with accessing the cluster.